from. _Error import Error


class CodaVuotaError(Error):
    pass
